### PR TITLE
docs: drop references to applyPolyfills as it is no longer needed for current target browsers

### PR DESCRIPTION
**Pull request by jcfranco**
cc @macandcheese

### CODE REVIEW

File: ember/README.md [hunk 1]
```diff
@@ -51,19 +51,12 @@ The best place to define the calcite-components is in an initializer.
...
-  defineCustomElements(window, {
-    resourcesUrl: "assets/calcite/"
-  });
+import {defineCustomElements} from '@esri/calcite-components/dist/loader';
 
+// define calcite components' custom elements on the window
+// define the resourcesUrl as well
+defineCustomElements(window, {
+  resourcesUrl: "assets/calcite/"
 });
```
Comment (benelan) [inline ember/README.md:60]: You're okay with us using `resourcesUrl` now? I remember you said to use `setAssetPath` since it is documented.

**Review by benelan — APPROVED** [review]
LGTM!

File: ember/README.md (hunk 1 above)
Comment (jcfranco) [reply ember/README.md:60]: I did a quick update, but using `setAssetPath` would be the right choice.

**Comment by jcfranco** [comment]
Closing in favor of https://github.com/Esri/calcite-components-examples/pull/67

**Pull request closed without merging**